src/main/java/seedu/address/logic/commands/schedule/EditScheduleCommand.java

```diff
@@ -26,7 +26,9 @@
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Edits the interview identified by the index number in the interview schedule "
             + "to the proposed date and time.\n"
-            + "Example: " + COMMAND_WORD + " 3 at/20-03-2023 10:00";
+            + "Parameters: INDEX at/DATE_TIME (in dd-MM-yyyy HH:mm format)\n"
+            + "Example: " + COMMAND_WORD + " 3 at/20-03-2023 10:00\n"
```

Comment:
Hi! Sorry I think I was supposed to write it in but.. maybe instead of `INDEX` we use `INTERVIEW_INDEX` for further clarity?